packages/bot-skeleton/src/scratch/hooks/css.js

```diff
@@ -372,7 +373,7 @@ Blockly.Css.CONTENT = [
     '}',
 
     '.blocklyText {',
-    'fill: var(--text-general);',
+    'fill: #333333;',
```

Comment:
Can `$color-black-1` be added in dark theme declaration so we can handle it from there instead of hard coding here?

Reply:
explanation in the next comment.